cleanlab/classification.py

```diff
@@ -557,6 +559,12 @@ def find_label_issues(
         <cleanlab.classification.CleanLearning.fit>`. See there for
```

Comment:
Move this into the Parameters block, just below the `save_space`

Reply:
can you clarify "this"

Reply:
addressed it partially, feel free to implement a better solution directly if you have one.